targets/targets.json

```diff
@@ -8172,11 +8172,15 @@
         }
     },
     "CY8CKIT_062_WIFI_BT_PSA": {
-        "inherits": ["NSPE_Target", "CY8CKIT_062_WIFI_BT"],
-        "extra_labels_add": ["PSA", "MBED_SPM"],
+        "inherits": ["NSPE_Target", "MCU_PSOC6_M4"],
```

Comment:
please restore the previous statae
PSA targets should always inherit from the closest match to the real target, for maintenance reasons

Reply:
Fixed in the https://github.com/ARMmbed/mbed-os/pull/10308/commits/7239375953203437a4c880ac65623ddbebf0a093